broker-daemon/utils/logger.js

```diff
@@ -28,6 +28,7 @@ if (process.env.NODE_ENV !== 'production') {
   logger.add(new winston.transports.Console({
     format: winston.format.combine(
       winston.format.timestamp(),
+      winston.format.colorize(),
```

Comment:
just as a note, this makes it harder to parse the output of logs (it adds extra characters to the output)